src/plugincontainer.cpp

```diff
@@ -769,6 +781,193 @@ const PreviewGenerator &PluginContainer::previewGenerator() const
   return m_PreviewGenerator;
 }
 
+void PluginContainer::simulateStartup(const std::vector<IPlugin*>& plugins) const

```

Comment:
So this repeats stuff that's scattered all over MO, right? Is there any way this can be made to _not_ simulate anything, but to actually be called when loading the plugins?

Reply:
Marking as resolved, comment on the new commits if you feel like it's not.